src/ng/templateRequest.js

```diff
@@ -86,7 +88,7 @@ function $TemplateRequestProvider() {
           cache: $templateCache,
           transformResponse: transformResponse
         }, httpOptions))
-        ['finally'](function() {
+        .finally(function() {
```

Comment:
>
> In src/ng/templateRequest.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72949259>:
>
> > @@ -86,7 +88,7 @@ function $TemplateRequestProvider() {
> >            cache: $templateCache,
> >            transformResponse: transformResponse
> >          }, httpOptions))
> > -        ['finally'](function() {
> > +        .finally(function() {
>
> This is intentional to make it possible for people to support some older
> browsers. Although the officially supported browsers aren't affected by
> this, people are able to support older browsers (e.g. older Androids) using
> polyfills and stuff like that, but .finallys were causing them problems.
> So, we had decided (iirc) to make their lives easier (even if we don't
> actively support such browsers), since it is a trivial change and doesn't
> affect anything else.
>
We already have lots of `.catch` occurrences in `src/` and Android 2.3
fails on that in exactly the same way. So it's already completely broken
there (parse errors === nothing works) at this very moment.